tests/src/Functional/ContentPersonRenderTest.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_whitelabel\Functional;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\node\NodeInterface;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\TestFileCreationTrait;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests that the Person content type renders correctly.
+ */
+class ContentPersonRenderTest extends WhitelabelBrowserTestBase {
+
+  use MediaTypeCreationTrait;
+  use TestFileCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'block',
+    'oe_whitelabel_starter_person',
+  ];
+
+  /**
+   * Tests the person page.
+   */
+  public function testPersonPage(): void {
+    $node = $this->createExamplePerson();
```

Comment:
lets do first a node with just the required fields, assert that labels (and any other elements?) that should not show with empty values don't.